app/services/item_change_set_persister.rb

```diff
@@ -43,6 +50,20 @@ def update_structural(updated)
     updated.new(structural: updated_structural)
   end
 
+  def access_changed?
+    copyright_statement_changed? || license_changed? || use_statement_changed?
+  end
+
+  def updated_access(updated)
+    access_properties = {
+      copyright: copyright_statement_changed? ? copyright_statement : updated.access.copyright,
+      license: license_changed? ? license : updated.access.license,
+      useAndReproductionStatement: use_statement_changed? ? use_statement : updated.access.useAndReproductionStatement
+    }.compact
```

Comment:
This is dangerous. If a new access property is added to cocina model in the future, this will delete it.

Reply:
Here is a safer pattern for the update:
```
  def updated_access(updated)
    access_properties = updated.access&.to_h || {}
    access_properties[:copyright] = copyright_statement if copyright_statement_changed?
    access_properties[:license] = license if license_changed?
    access_properties[:useAndReproductionStatement] = use_statement if use_statement_changed?
    update.new(access: access_properties.compact.presence)
 end
```

Reply:
@justinlittman Hmmm, are you sure it'll delete other properties? I thought creating an instance of an existing dry-struct model with a subset of properties would pass through unspecified properties. For instance:

```ruby
# First, create a dummy access instance.
[1] pry(main)> access = Cocina::Models::DROAccess.new
=> #<Cocina::Models::DROAccess access="dark" controlledDigitalLending=nil copyright=nil embargo=nil download="none" readLocation=nil useAndReproductionStatement=nil license=nil>
# Then, create a new instance based on the existing one changing only the copyright. Only the copyright changes in the new instance, passing through all other properties.
[2] pry(main)> access.new(copyright: 'foo')
=> #<Cocina::Models::DROAccess access="dark" controlledDigitalLending=nil copyright="foo" embargo=nil download="none" readLocation=nil useAndReproductionStatement=nil license=nil>
# Finally, create a new instance based on the existing one changing the copyright and the license. Only the copyright and license change in the new instance, again passing through all other properties.
[3] pry(main)> access.new(copyright: 'foo', license: 'bar')
=> #<Cocina::Models::DROAccess access="dark" controlledDigitalLending=nil copyright="foo" embargo=nil download="none" readLocation=nil useAndReproductionStatement=nil license="bar">
```